driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -113,6 +116,22 @@ void setListenAddress(InetAddress listenAddress) {
         this.listenAddress = listenAddress;
     }
 
+    void setDSEVersion(String dseVersion) {
```

Comment:
There is a slight inconsistency here in that we use mixed case for getters and upper case for setters for 'DSE' (`getDseVersion` vs. `setDSEVersion`),we should make consistent and use mixed case.